macosx/InfoActivityViewController.mm

```diff
@@ -122,39 +121,36 @@ - (void)checkLayout
         self.fActivityStackView.orientation = NSUserInterfaceLayoutOrientationVertical;
         self.fActivityStackView.spacing = kStackViewVerticalSpacing;
         self.fCurrentHeight = self.fVertLayoutHeight;
+        isVertLayout = YES;
     }
+    
+
+    return isVertLayout;
```

Comment:
I don't think this was my suggestion :) Sorry if I wasn't clear enough. What I had in mind was something along the lines of

```cpp
auto const oldOrientation = self.fActivityStackView.orientation;
if (...) { ... } else { ... }
return oldOrientation != self.fActivityStackView.orientation;
```

Otherwise what you have now will only lead to layout update if the new layout is vertical, not if the layout has changed, including from vertical to horizontal.

Reply:
@mikedld is this resolved?

Reply:
Indirectly since calls to `updateWindowLayout` are now unconditional.